woocommerce/woocommerce-android
# Subscription events

Closes: #8563 


### Description
This PR adds analytics tracks to the usage of the subscription extension read-only support within the app.
* `order_details_subscriptions_shown` [event registration](https://github.com/Automattic/tracks-events-registration/pull/1474)
* `product_details_view_subscriptions_tapped` [event registration](https://github.com/Automattic/tracks-events-registration/pull/1475)

### Testing instructions
TC1
1. Open the Orders List Screen
2. Open an order containing a subscription
3. Check that the subscription information is shown
4. Check that the `order_details_subscriptions_shown` event is sent

TC2
1. Open the Products List Screen
2. Open a subscription product details
3. Check that the subscription card is shown
4. Tap on the subscription card
5. Check that subscription details are shown
6. Check that the `product_details_view_subscriptions_tapped` event is sent

### Images/gif
```
🔵 Tracked: order_details_subscriptions_shown, Properties: {"blog_id":214253715,"is_wpcom_store":true,"is_debug":true}

🔵 Tracked: product_details_view_subscriptions_tapped, Properties: {"blog_id":214253715,"is_wpcom_store":true,"is_debug":true}
```
